### PR TITLE
Add Hefe filter

### PR DESCRIPTION
**Pull request by shelune**

### CODE REVIEW

File: source/scss/hefe.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  filter: brightness(1.1) contrast(1.1) sepia(.1) saturate(1.15);
+
+  &::after {
+    box-shadow: inset 5px -5px 20px rgba(21, 12, 0, .7);
+  }
+
+  &::before {
+    background: rgba(67, 45, 31, .7);
+    mix-blend-mode: overlay;
+    box-shadow: inset -5px 5px 20px rgba(21, 12, 0, .7);
```
Comment (una) [inline source/scss/hefe.scss]: i'd use a percentage here for box shadow because the image may change size

**Comment by una** [comment]
Please only include hefe file

**Comment by shelune** [comment]
Sorry I'm still getting used to using git, please check it again.

File: source/scss/hefe.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  filter: brightness(1.1) contrast(1.1) sepia(.1) saturate(1.15);
+
+  &::after {
+    box-shadow: inset .3rem -.3rem 20px rgba(21, 12, 0, .7);
+  }
+
+  &::before {
+    background: rgba(67, 45, 31, .7);
+    mix-blend-mode: overlay;
+    box-shadow: inset -.3rem .3rem 20px rgba(21, 12, 0, .7);
```
Comment (una) [inline source/scss/hefe.scss]: please use ems or a percentage instead of rems for consistency (percent probably makes more sense as image sizes change)

**Comment by una** [comment]
That's okay! It's a complex system to get used to at first :smile:

File: source/scss/hefe.scss (hunk 2 above)
Comment (shelune) [inline source/scss/hefe.scss]: Is it ok to use em?  I haven't figured how to use % in box-shadow and em is responsive too :)

**Comment by una** [comment]
Hey, should we close this because it got a bit outdated? Maybe try resubmitting?

**Pull request closed without merging**